WebDevStudios/nextjs-wordpress-starter
# Feature/149 card

Closes https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/149

### Link

https://nextjs-wordpress-starter-8g440yohi.vercel.app/

### Description

* Add card component
* Add heading component
* Add cards to the blog and team archive pages

### Screenshot

![ 2021-02-03 at 13 08 48 ](https://user-images.githubusercontent.com/12365909/106790100-122fab80-6621-11eb-8cbf-086857ddae43.png)
![ 2021-02-03 at 13 09 15 ](https://user-images.githubusercontent.com/12365909/106790106-13f96f00-6621-11eb-8186-2071aeb5d61f.png)


### Verification

How will a stakeholder test this?

1. Checkout the branch
2. Run storybook
3. Make sure the card works
4. Run the app and navigate to the blog and team members archive pages
5. Ensure the cards render in the post grid
